### ISSUE TITLE
empty value not evaluating to default as expected

### ISSUE THREAD

**Issue by samthomson** [s-yadav/react-number-format#316]
If I change the text input to be blank, I expect the value to be what I set the default value to. But it is undefined?
How can I achieve my intended behavior? I would like that when the user clears the text input, it sets the value of `0` to my local `nMinSpread` variable.

![image](https://user-images.githubusercontent.com/6588340/58434835-2e89ef80-80ad-11e9-8cb8-427b2e68bd65.png)

**Comment by Grsmto** [comment]
I don't think this is "expected". A default value is not the same as an empty state. A default **is** a value. Whereas an empty state isn't.
You could do that yourself with a condition in the `onValueChange` function, I don't think the library should do that for you.
